app.js

```diff
@@ -0,0 +1,17 @@
+const express = require("express")
+const app = express()
+const { getTopics, getArticles } = require("./controlllers/ncNewsController")
+const {handle404Statuses, handle500Statuses} = require("./controlllers/errorHandlingController")
+app.use(express.json())
+
+app.get("/api/topics", getTopics)
+app.get("/api/articles", getArticles)
+
+app.all('*', (req, res, next)=>{
+    next({status:404})
```

Comment:
Think you're passing this into your handle404Statuses below with your next - but you can handle generic "bad pathways" with just this app.all by sending back res.status(404).send({msg: "404 bad pathway"}) or similar.